### ISSUE TITLE
--valign-exclusion-list="q" does not preserve previous import quote list alignment

### ISSUE THREAD

**Issue by ugexe** [perltidy/perltidy#101]
Previously the following could keep its current alignment:

```
use Baz qw( ooo )
use Foo      ();
use Foo::Bar ();
```

now by default that gets tidied into:

```
use Baz      qw( ooo )
use Foo      ();
use Foo::Bar ();
```

The docs suggest that `--valign-exclusion-list="q"` would use the old alignment behavior, but it results in the following (instead of what is shown in the first example):

```
use Baz qw( ooo )
use Foo ();
use Foo::Bar ();
```

**Comment by shancock9** [comment]
Yes, this was a compromise to keep the controls simple.  The problem is that it is very reasonable to put something like


```
use Foo ();
```

and

```
use Foo qw();
```

in the same vertical alignment group (type 'q') since they are functionally equivalent.  That is the way it is now.  This adds the ability to turn off alignment of empty parens, which was not previously possible.

It would not be difficult to add a flag which gives the option of either linking or not linking () and qw together, and in fact I have a working prototype which does that. But I'm not sure if the added complexity of yet another parameter is worthwhile. A blank line can always be inserted to break unwanted alignment.

**Comment by ugexe** [comment]
>  This adds the ability to turn off alignment of empty parens, which was not previously possible.

I think I'm confused... which flag disables alignment of empty parens? In my first pre-tidied example the two imports with empty parens have them aligned manually. But the 3rd example in which the `--valign-exclusion-list="q"` flag (the flag I'm assuming you are referring to) is used has the paren alignment changed.

**Comment by shancock9** [comment]
In previous versions, it was not possible to control the alignment of empty parens in sequence of 'use' statements - they always got aligned. So in your first example, you would not have been able to prevent the alignment of the empty parens. This was not always wanted, and sometimes people fenced off sections of code with ``#<<< comments`` to avoid this alignment.  

In the current version, empty parens and qw quotes in 'use' statements are lumped into an alignment group named 'q'. So -vxl='q' turns of the alignment of any consecutive sequence of members of this group, as in your third example.

I appreciate the problem that changes in formatting cause and try to minimize them, and minimize the frequency of releases.

The next release of perltidy will probably be towards the end of the year. But if you think it is worthwhile, I could make a tagged version at github which gives independent control of qw quotes and empty parens. The control would be something like this:

```
--valign-empty-parens-with-qw, or -vpq
```

This would be on by default and puts empty parens and qw quotes in the same vertical alignment group 'q'.

Setting -nvpq would put empty parens in their own separate alignment group, probably named '()'.  But qw quotes would still be in alignment group 'q'.  So these two groups would be separately controlled,  with independent vertical alignment.  They could be turned off independently with -vxl='q' and -vxl='()'.

So to recover the previous behavior, the command would be:

```
-nvpq -vxl='q'
```

The first parameter separates () and qw into separate alignment groups. The second parameter stops alignment of qw quotes, but () items remain aligned.

I have a prototype version which does this but am reluctant to implement it because I get a lot of complaints about the complexity of perltidy parameters.
